.github/workflows/publish.yml

```diff
@@ -19,8 +20,9 @@ jobs:
           scope: "@bucketco"
       - name: Install dependencies
         run: yarn install --immutable
-      - name: Build SDK
+      - name: Build packages
         run: yarn build
-      - run: npm publish
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
just checking, but this'll only publish if there's been a version change, right? I guess any attempt at publishing will fail if there's no new version anyway.

Reply:
Yep exactly. I just figured I'd disable until I make a version PR later today so I can iron out any possible issues then